kbase-extension/scss/partials/_jobLog.scss

```diff
@@ -1,123 +1,130 @@
-@use "sass:color";
+.kb-job-log__container {
+  background-color: use-color('info-lighter');
+  padding: 1rem;
+}
 
-#kblog {
-  /* #kblog-header */
-  &-header {
-    margin: 5px 0;
-  }
+%line_container {
+  @include fixed-width;
 
-  /* #kblog-panel */
-  &-panel {
-    overflow: auto;
-    height: 350px;
-    padding: 5px;
-    border: 1px solid rgb(206 206 206);
-    background-color: rgb(245 245 245);
-    color: rgb(0 0 0);
-    resize: vertical;
-    vertical-align: top;
+  &:first-child {
+    padding-top: 1rem;
   }
 
-  /* #kblog-err, #kblog-msg */
-  &-err,
-  &-msg {
-    margin-top: 5px;
+  &:last-child {
+    padding-bottom: 1rem;
   }
 }
 
-.kblog,
+%line_number {
```

Comment:
`%name` is a placeholder selector, which doesn't appear in the css output but which is useful for storing sets of css attributes. See https://sass-lang.com/documentation/style-rules/placeholder-selectors